src/metabase/channel/email/messages.clj

```diff
@@ -120,7 +120,7 @@
   [email sso-source password-reset-url is-active?]
   {:pre [(u/email? email)
          ((some-fn string? nil?) password-reset-url)]}
-  (let [google-sso? (= "google" sso-source)
```

Comment:
Looks like this is fixing an old bug right? We transformed `:sso_source` to a keyword but didn't change this?

Reply:
yeah this was a pre-existing bug, just noticed when I wrote tests I was getting the generic SSO email, but it looks like this change might have broken some other test so I'll check that out.